clients/client-java/src/main/java/com/datastrato/gravitino/client/GravitinoAdminClient.java

```diff
@@ -188,7 +190,7 @@ public GravitinoAdminClient build() {
       Preconditions.checkArgument(
           uri != null && !uri.isEmpty(), "The argument 'uri' must be a valid URI");
 
-      return new GravitinoAdminClient(uri, authDataProvider);
+      return new GravitinoAdminClient(uri, authDataProvider, headers);
```

Comment:
I'm not sure whether this should be named `headers`. Because it may be weird to have headers for a Client. HttpClient has `properties` that have not been used yet. I don't know why we need headers and properties simultaneously.

Reply:
Does the header work on every request?

Reply:
I think so. It will work on every request.

Reply:
What's the meaning of `properties` here, I think it is unrelated to headers here?

Reply:
> Maybe we can reuse `properties` and apply some properties that start with `header.` to the request header

If the key starts with `header.` we add it to the request header and others to the properties. Do you think this is OK?

Reply:
> What's the meaning of `properties` here, I think it is unrelated to headers here?

I think it may be some client-side configurations, such as connection pool size, timeout parameters, etc., but we haven't started using it yet.

Reply:
So do we want just one `withProperties()` method to pass the `header.xxx` to `header` and the others to `properties`? Or `withProperties()` method to set the `properties` and `withHeaders` method to set the `headers`

Reply:
We don't have to mix headers with properties, I'm OK with the current implementation.